pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -380,20 +365,16 @@ def __process_eval_epoch_end_results_and_log_legacy(self, eval_results):
                 eval_results = [eval_results]
 
             num_loaders: int = self.trainer.evaluation_loop.num_dataloaders
-            prog_bar_metrics, log_metrics, callback_metrics = {}, {}, {}
+            prog_bar_metrics, log_metrics = {}, {}
 
             for result_idx, result in enumerate(eval_results):
-                _, prog_bar_metrics, log_metrics, callback_metrics, _ = self.trainer.process_dict_result(result)
+                _, prog_bar_metrics, log_metrics, _ = self.trainer.process_dict_result(result)
```

Comment:
What are the extra returned objects ?

Reply:
`output, progress_bar_metrics, log_metrics, hiddens`

To be removed in separate PRs